app/domain/payment_providers/z54.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, Die Mitte Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+# rubocop:disable Metrics/MethodLength, Metrics/LineLength, Metrics/AbcSize
+class PaymentProviders::Z54 < Epics::GenericRequest
+
+  attr_accessor :from, :to
+
+  def initialize(client, from = nil, to = nil)
+    super(client)
+    self.from = from
+    self.to = to
+  end
+
+  def header
+    Nokogiri::XML::Builder.new do |xml|
+      xml.header(authenticate: true) {
+        xml.static {
+          xml.HostID host_id
+          xml.Nonce nonce
+          xml.Timestamp timestamp
+          xml.PartnerID partner_id
+          xml.UserID user_id
+          xml.Product("EPICS - a ruby ebics kernel", 'Language' => 'de')
+          xml.OrderDetails {
+            xml.OrderType 'Z54'
+            xml.OrderAttribute 'DZHNN'
+            if !!from && !!to
```

Comment:
warum das Umwandeln in ein `boolean`? Wenn nur das vorhandensein geprüft wird, braucht es das IMHO nicht.